paddle/framework/backward.md

```diff
@@ -2,7 +2,7 @@
 
 ## Motivation
 
-In Neural Network, many model is solved by the the backpropagation algorithm(known as BP) at present. Technically it caculates the gradient of the loss function, then distributed back through the networks. Follows the chain rule, so we need a module chains the gradient operators/expressions together with to construct the backward pass. Every forward network needs a backward network to construct the full computation graph, the operator/expression's backward pass will be generated respect to forward pass. 
+In Neural Network, most models are solved by the the backpropagation algorithm(known as BP) at present. Technically, it calculates the gradient of the loss function, then propagates it back through the networks following the chain rule. Hence we need a module that chains the gradient operators/expressions together to construct the backward pass. Every forward network needs a backward network to construct the full computation graph. The operator/expression's backward pass will be generated with respect to the forward pass. 
```

Comment:
At the moment, most neural network models are solved by the the backpropagation algorithm (known as **BP**). Technically, BP calculates the gradient of the loss function and propagates the gradient back through the network, following the chain rule. Hence we need a module that chains the gradient operators or expressions together to construct the backward pass. Every forward network needs a backward network to construct the full computation graph. The operator or expression's backward pass will be generated with respect to the forward pass.